src/components/TabSelector/TabSelector.tsx

```diff
@@ -27,12 +27,13 @@ type TabSelectorProps = MaterialTopTabBarProps & {
 type IconAndTitle = {
     icon: IconAsset;
     title: string;
+    testID?: string;
 };
 
 function getIconAndTitle(route: string, translate: LocaleContextProps['translate']): IconAndTitle {
     switch (route) {
         case CONST.TAB_REQUEST.MANUAL:
-            return {icon: Expensicons.Pencil, title: translate('tabSelector.manual')};
```

Comment:
Is there another way we could do this? I don't love updating this function only in this certain case...

Reply:
Let me check it

Reply:
We need to mark the elements we're waiting for before rendering so we can manipulate them further.
Instead of hardcoding a testID for just one case, we could generate it dynamically based on the route, but that would add unnecessary complexity and feel wrong. 
Since we'll likely be adding more E2E tests, this generic function, which creates components for multiple screens, will also assign an ID to each element

Reply:
can we maybe just update the function to `getIconTitleAndTestID` and have it return a testID (even if unused) for all options? I think that would make me feel better haha